spec/models/organization_spec.rb

```diff
@@ -6,4 +6,14 @@
   end
 
   it_should_behave_like 'notable'
+
+  it 'errors when primary_contact does not belong to organization' do
+    contact = create(:person)
+    expect{ create(:organization, primary_contact: contact) }.to raise_error(ActiveRecord::RecordInvalid)
+  end
+
+  it 'primary_contact can be added when also part of organization' do
+    contact = create(:person)
+    create(:organization, primary_contact: contact, people: [contact])
+  end
```

Comment:
Nice specs. You could DRY them up a bit by:

1. Putting them in a `describe 'primary_contact'` block
1. Moving `contact` to a `let` block inside the `describe`

I would also please put the second `create` into an `expect { ... }.not_to raise_error` to clarify intent.